internal/command/create/create.go

```diff
@@ -34,10 +34,12 @@ func (c *Command) Run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+
+	c.generateGoMod(ctx, tmpDir)
```

Comment:
Let's check for the error and return the error if there's a failure.

Reply:
Updated